src/fmu/dataio/datastructure/meta/content.py

```diff
@@ -20,65 +20,72 @@
 
 
 class Timestamp(BaseModel):
-    """
-    Time stamp for data object.
-    """
+    """A timestamp object contains a datetime representation of the time
+    being marked and a string label for this timestamp."""
 
     label: Optional[str] = Field(
         default=None,
         examples=["base", "monitor", "mylabel"],
     )
+    """A string label corresponding to the timestamp."""
+
     value: Optional[Union[NaiveDatetime, AwareDatetime]] = Field(
         default=None,
         examples=["2020-10-28T14:28:02"],
     )
+    """A datetime representation."""
 
 
 class Time(BaseModel):
+    """A block containing lists of objects describing timestamp information for this
+    data object, if applicable, like Flow simulator restart dates, or dates for seismic
+    4D surveys.  See :class:`Time`.
+    .. note:: ``data.time`` items can currently hold a maximum of two values."""
+
     t0: Optional[Timestamp] = None
+    """The first timestamp. See :class:`Timestamp`."""
+
     t1: Optional[Timestamp] = None
+    """The second timestamp. See :class:`Timestamp`."""
 
 
 class Seismic(BaseModel):
     """
-    Conditional field
+    A block describing seismic data. Shall be present if ``data.content``
+    == ``seismic``.
     """
 
-    attribute: Optional[str] = Field(
-        default=None,
-        examples=["amplitude_timeshifted"],
-    )
-    calculation: Optional[str] = Field(
-        default=None,
-        examples=["mean"],
-    )
-    filter_size: Optional[float] = Field(
-        allow_inf_nan=False,
-        default=None,
-    )
-    scaling_factor: Optional[float] = Field(
-        allow_inf_nan=False,
-        default=None,
-    )
-    stacking_offset: Optional[str] = Field(
-        default=None,
-        examples=["0-15"],
-    )
-    zrange: Optional[float] = Field(
-        allow_inf_nan=False,
-        default=None,
-    )
+    attribute: Optional[str] = Field(default=None, examples=["amplitude_timeshifted"])
+    """A known seismic attribute."""
+
+    calculation: Optional[str] = Field(default=None, examples=["mean"])
+    """The type of calculation applied."""
+
+    filter_size: Optional[float] = Field(allow_inf_nan=False, default=None)
+    """The filter size applied."""
+
+    scaling_factor: Optional[float] = Field(allow_inf_nan=False, default=None)
+    """The scaling factor applied."""
+
+    stacking_offset: Optional[str] = Field(default=None, examples=["0-15"])
+    """The stacking offset applied."""
+
+    zrange: Optional[float] = Field(allow_inf_nan=False, default=None)
+    """The z-rage of these data."""
```

Comment:
```suggestion
    """The z-range of these data."""
```

Reply:
I guess we don't want to store the emotional state of the data...